server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/TasksRoutes.java

```diff
@@ -196,7 +196,7 @@ private Duration getTimeout(Request req) {
                 .filter(parameter -> !parameter.isEmpty());
 
             requestTimeout.ifPresent(timeout ->
-                Preconditions.checkState(!timeout.replaceAll(" ", "").startsWith("-"), "Timeout should not be positive"));
+                Preconditions.checkState(!timeout.replaceAll(" ", "").startsWith("-"), "Timeout should be positive"));
```

Comment:
Checking the first character is a strange way of ensuring a number is negative...

(Sorry, I missed it so far)

Reply:
It's the only way I've founded to check it, since an `Integer.parseXXX` would not recognize the unit and duration parsing won't recognize the minus sign, throwing a generic error

Reply:
Let me have a look...

Reply:
See https://github.com/linagora/james-project/pull/2653/commits/2f10137516980ab1b8c39b7a20722cc31d0ca2e8

Reply:
Thank you